community/btop/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make STATIC=true ADDFLAGS="$CXXFLAGS"
```

Comment:
Might want to make `ADDFLAGS="$CFLAGS $CXXFLAGS"` not just `$CXXFLAGS`.

Reply:
Happily, thanks for the review